MaterialLib/SolidModels/CreepBGRa.cpp

```diff
@@ -84,7 +84,11 @@ CreepBGRa<DisplacementDim>::integrateStress(
     if (!success_iterations)
         return {};
 
-    KelvinMatrix tangentStiffness = linear_solver.solve(C);
+    // If *success_iterations>0, tangentStiffness = J_(sigma)^{-1}C
+    // where J_(sigma) is the Jacobian of the last local Newton-Raphson
+    // iteration, which is already LU decomposed.
+    KelvinMatrix tangentStiffness =
+        (*success_iterations == 0) ? C : linear_solver.solve(C);
```

Comment:
can `success_iterations == 0` ever happen?

Reply:
Yes. If I remember correctly, in the first time step only, somehow the residual is 0 leading to 0 iterations...

Reply:
Yes, it happens when ||r^0||=0. For example with isotropic initial stress , in the case of the first time step and the first global Newton step, we have ||s||=0 and ||d\sigma| =0 (an elasticity status), meaning ||r||=0.

Reply:
Thanks for the explanation